runtime/src/main/java/org/corfudb/runtime/exceptions/OverwriteException.java

```diff
@@ -1,7 +1,15 @@
 package org.corfudb.runtime.exceptions;
 
+import lombok.Getter;
+
 /**
  * Created by mwei on 12/14/15.
  */
+@Getter
 public class OverwriteException extends LogUnitException {
+   private OverwriteCause overWriteCause;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "overWriteCause" field final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1165)